MvvmCross/Core/Core/ViewModels/MvxViewModelExtensions.cs

```diff
@@ -32,6 +32,10 @@ public static void CallBundleMethods(this IMvxViewModel viewModel, string method
         public static void CallBundleMethod(this IMvxViewModel viewModel, MethodInfo methodInfo, IMvxBundle bundle)
         {
             var parameters = methodInfo.GetParameters().ToArray();
+
+            if (bundle == null && parameters.Count() > 0)
```

Comment:
Because we have `public async Task Init(string parameter)` in the MvxViewModel<TParameter> class. CallBundleMethod will try to call this, but the bundle is null. So when the bundle is null it should only call Init methods without parameters.